app/components/forms/user-payment-info-form.js

```diff
@@ -0,0 +1,92 @@
+import Component from '@ember/component';
+import FormMixin from 'open-event-frontend/mixins/form';
+import { validPhoneNumber } from 'open-event-frontend/utils/validators';
+import { computed } from "@ember/object";
```

Comment:
@uds5501 use single quotes - `'@ember/object'

Reply:
@shreyanshdwivedi thanks for pointing out, fixed it now!